_health-care/_js/reducers/uiState/index.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+import lodashDeep from 'lodash-deep';
+
+import { SECTION_COMPLETE_UPDATE_FIELD, SECTION_REVIEW_STATUS_UPDATE_FIELD } from '../../actions';
+
+// Add deep object manipulation routines to lodash.
+_.mixin(lodashDeep);
+
+
+const ui = {
+  completedSections: {
```

Comment:
I was calling it `completedSections` because that is the state of the UI. Also, the purpose of `uiState` is to give people another place to put UI related state changes so it may not always be just `completedSections`.